cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EnergyConsumerConversion.java

```diff
@@ -40,12 +40,48 @@ public void convert() {
                 .setLoadType(loadType);
         identify(adder);
         connect(adder);
+        model(adder);
         Load load = adder.add();
         addAliasesAndProperties(load);
         convertedTerminals(load.getTerminal());
         setLoadDetail(loadKind, load);
     }
 
+    private void model(LoadAdder adder) {
+        p.asBoolean("exponentModel").ifPresent(exponentModel -> {
+            if (Boolean.TRUE.equals(exponentModel)) {
+                double pVoltageExponent = p.asDouble("pVoltageExponent", 0);
+                double qVoltageExponent = p.asDouble("qVoltageExponent", 0);
+                boolean constantPower = pVoltageExponent == 0 && qVoltageExponent == 0;
+                if (!constantPower) {
+                    adder.newExponentialModel()
+                            .setNp(pVoltageExponent)
+                            .setNq(qVoltageExponent)
+                            .add();
+                }
+            } else {
+                double pConstantCurrent = p.asDouble("pConstantCurrent");
+                double pConstantImpedance = p.asDouble("pConstantImpedance");
+                double pConstantPower = p.asDouble("pConstantPower");
+                double qConstantCurrent = p.asDouble("qConstantCurrent");
+                double qConstantImpedance = p.asDouble("qConstantImpedance");
+                double qConstantPower = p.asDouble("qConstantPower");
```

Comment:
Are you sure 0 is an officiel default value? I saw in a CGMES doc that depending of exponentModel they are all required. Furthermore, it should also be required that there sum is equals to 1 so it is dangerous to let all of them optional and have a default value.

Reply:
Not sure at all!
The documentation states that indeed these attributes must be specified but: `In case the values for LoadResponseCharacteristic cannot be obtained, the default values for
pConstantPower and qConstantPower are set to 1.` so maybe you can check if every value is present and if at least one is missing we do the default values (pConstantPower and qConstantPower 1) and ignore all others?

@zamarrenolm I don't know if you have an opinion about it

Reply:
@zamarrenolm can you check that at least all IGMs are robust to this and do not crash? 
@geofjamg just to be safe, I would prefer to ignore everything if data is missing rather than wait if it crashes

Reply:
Yes, we should avoid regression as we have no idea of the overall quality for these specific data as until now we were ignoring its. So we check that the 3 coefficients (for active and reactive part) have been defined and maybe also check that sum is one (because we would get an exception for the IIDM adder) and only create the load model in that case?

Reply:
For the sum, we can ratio if it does not equals to 1? It is what we do for `CoordinatedReactiveControl` for example

Reply:
> @zamarrenolm can you check that at least all IGMs are robust to this and do not crash? @geofjamg just to be safe, I would prefer to ignore everything if data is missing rather than wait if it crashes

We have confirmed that the current implementation does not crash with available IGMs: it correctly reads and maps to the IIDM ZIP model the CGMES load response characteristic for the two TSOs that define loads with non constant power consumption.